UnitTests/GitCommandsTests/Git/Gpg/GitGpgControllerTests.cs

```diff
@@ -47,5 +49,144 @@ public void Validate_GetRevisionCommitSignatureStatusAsync_good_signature(Commit
 
             Assert.AreEqual(expected, actual);
         }
+
+
+        [TestCase]
+        public void Validate_GetRevisionCommitSignatureStatusAsync_null_revision()
+        {
+            ExecuteGetRevisionCommitSignatureStatusAsync_null(null);
```

Comment:
this is overly complicated. it can be simplified to one line:

```
        [TestCase]
        public void Validate_GetRevisionCommitSignatureStatusAsync_null_revision()
        {
            ((Func<Task>)(async () => await _gpgController.GetRevisionCommitSignatureStatusAsync(null))).ShouldThrow<ArgumentNullException>();
        }
```

please refactor other tests to match

Reply:
For me it's not so complicated.
Looking at this "compact" way, the first time it looks a bit strange to see.
I change the tests.